<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-BTLCFE6" failures="1" tests="1" name="testcases.AddCustomerTest" time="0.435" errors="0" timestamp="2024-03-05T12:14:55 EET" skipped="0">
  <testcase classname="testcases.AddCustomerTest" name="addCustomer" time="0.435">
    <failure message="somereason" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: somereason
at org.testng.Assert.fail(Assert.java:111)
at testcases.AddCustomerTest.addCustomer(AddCustomerTest.java:23)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:228)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:961)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:201)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:819)
at org.testng.TestRunner.run(TestRunner.java:619)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1228)
at org.testng.TestNG.runSuites(TestNG.java:1134)
at org.testng.TestNG.run(TestNG.java:1101)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:377)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:138)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:465)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:451)
]]>
    </failure>
    <system-out>
      <![CDATA[Capturing a screenshot
Click to see Screenshot
<a target="_blank" href=Tue_Mar_05_12_14_54_EET_2024.jpg>Screenshot</a>
<br>
<br>
<a target="_blank" href=Tue_Mar_05_12_14_54_EET_2024.jpg><img src=Tue_Mar_05_12_14_54_EET_2024.jpg height=200 width=200></img></a>]]>
    </system-out>
  </testcase> <!-- addCustomer -->
  <system-out>
    <![CDATA[Capturing a screenshot
Click to see Screenshot
<a target="_blank" href=Tue_Mar_05_12_14_53_EET_2024.jpg>Screenshot</a>
<br>
<br>
<a target="_blank" href=Tue_Mar_05_12_14_53_EET_2024.jpg><img src=Tue_Mar_05_12_14_53_EET_2024.jpg height=200 width=200></img></a>
Capturing a screenshot
Click to see Screenshot
<a target="_blank" href=Tue_Mar_05_12_14_54_EET_2024.jpg>Screenshot</a>
<br>
<br>
<a target="_blank" href=Tue_Mar_05_12_14_54_EET_2024.jpg><img src=Tue_Mar_05_12_14_54_EET_2024.jpg height=200 width=200></img></a>]]>
  </system-out>
</testsuite> <!-- testcases.AddCustomerTest -->
